### PR TITLE
Add weekly highlights for the week ending July 1, 2022

### PR DESCRIPTION
**Pull request by marians**
Please add your contributions until July 1, 13:00 CEST.

### CODE REVIEW

File: src/content/changes/highlights/2022-07-01-md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+## Apps
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-07-01-md:13]:
```suggestion
## Apps
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.5.0](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#050---2022-06-22) introduces support for exporting CIS benchmark reports, and makes it possible to shard metrics across multiple instances of the exporter.

```

**Comment by gianfranco-l** [comment]
@ljakimczuk as agreed, could you add a brief description about out-of-band delivery?

**Comment by ljakimczuk** [comment]
@gianfranco-l sure:

The `out-of-band` delivery is a new feature introduced in the `gitops-template` that allows for delivering resources directly to the Workload Cluster with Flux, skipping the App Platform components.

**Comment by gianfranco-l** [comment]
added honeybadger contribution for this week

**Pull request merged**